providers/fips/fipsprov.c

```diff
@@ -727,12 +636,8 @@ int OSSL_provider_init(const OSSL_CORE_HANDLE *handle,
     if (!SELF_TEST_post(&selftest_params, 0))
         goto err;
 
-    /*
-     * TODO(3.0): Remove me. This is just a dummy call to demonstrate making
-     * EVP calls from within the FIPS module.
-     */
-    if (!dummy_evp_call(libctx))
-        goto err;
+    /* TODO(3.0): Tests will hang if this is removed */
```

Comment:
Huh?? I wonder why?

Reply:
It seems to hang during cleanup with a NULL read lock.. A double free maybe?